src/TodoAddBtn.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react'
+
+import PortalModal from './PortalModal'
+import SampleModal from './SampleModal'
+
+const TodoAddBtn = () => {
+  const [isModalOpen, isOpen] = useState(false)
```

Comment:
보통 state의 naming convention은 아래와 같습니다.
```suggestion
  const [isModalOpen, setIsModalOpen] = useState(false)
```

상태의 이름 맨 앞에 set을 붙이는게 관행이긴 해요